src/core/video/handlers/resize.handler.ts

```diff
@@ -0,0 +1,47 @@
+import type { AdapterParams } from '../../common/schemas';
+import { adapterParamsSchema } from '../../common/schemas';
+import { getAdapter } from '../../../adapters/adapter.utils';
+import type { FastifyInstance } from 'fastify';
+import { VideoService } from '../video.service';
+
+type ResizeQuery = { width: number; height: number, codecName: string };
+
+const resizeQuerySchema = {
+  type: 'object',
+  properties: {
+    height: { type: 'number' },
+    width: { type: 'number' },
+    codecName: { type: 'string' },
```

Comment:
@JonaszJestem Can u convince me this property is useful? From client's perspective, all I want to do is resize video, now I also have to know its codec.

Reply:
Resolved in #17